src/commands/logs/functions.mts

```diff
@@ -0,0 +1,101 @@
+import { Argument, OptionValues } from 'commander'
+import inquirer from 'inquirer'
+
+import { chalk, log } from '../../utils/command-helpers.mjs'
+import { getWebSocket } from '../../utils/websockets/index.mjs'
+import BaseCommand from '../base-command.mjs'
```

Comment:
Similar to earlier comments, there's a few imports here that can be imported using `import type` rather than just `import` since they're only being used for their type definitions